pgcli/packages/pgspecial/__init__.py

```diff
@@ -0,0 +1,12 @@
+__all__ = []
+
+def export(defn):
+    """Decorator to explicitly mark functions that are exposed in a lib."""
+    globals()[defn.__name__] = defn
+    __all__.append(defn.__name__)
+    return defn
+
+from .dbcommands import *
+from .iocommands import *
+
+__all__.append(iocommands.TIMING_ENABLED)
```

Comment:
I think it should be just TIMING_ENABLED here, since you're importing * from iocommands.